scripts/pymultic

```diff
@@ -223,14 +255,38 @@ def container_compile(ver, infile):
         os.unlink(outfile)
 
     print('*** Compiling for Python {}'.format(fullver))
-    proc = subprocess.Popen([container_exe, 'run'] + CONTAINER_EXE_EXTRA_ARGS[container_exe] +
-                            ['--rm', '--name', '{}'.format(outfile),
-                             '-v', '{}:/indir:Z'.format(indir),
-                             '-v', '{}:/outdir:Z'.format(os.getcwd()), '-w', '/outdir',
-                             'python:{}'.format(fullver),
-                             'python', '-c',
-                             "import py_compile; py_compile.compile('/indir/{}', '{}')".format(infile, outfile)])
-    proc.communicate()
+    if ver in {'1.0', '1.1', '1.2', '1.3', '1.4'}:
+        # The hard way -- hope your code is safe...
+        srcdir = os.path.dirname(os.path.realpath(infile))
+        comptmp = os.path.join(indir, 'pymc_temp.py')
+        if os.path.exists(comptmp):
+            os.unlink(comptmp)
+        shutil.copyfile(infile, comptmp)
+        proc = subprocess.Popen([container_exe, 'run'] + CONTAINER_EXE_EXTRA_ARGS[container_exe] +
+                                ['--rm', '--name', '{}'.format(outfile),
+                                 '-v', '{}:/indir:Z'.format(indir),
+                                 '-v', '{}:/outdir:Z'.format(os.getcwd()), '-w', '/outdir',
+                                 'python:{}'.format(fullver),
+                                 'python', '-c',
+                                 "import pymc_temp"])
+        proc.communicate()
+        if os.path.exists(comptmp + 'o'):
+            shutil.copyfile(comptmp + 'o', outfile)
+            os.unlink(comptmp + 'o')
+        elif os.path.exists(comptmp + 'c'):
+            shutil.copyfile(comptmp + 'c', outfile)
+            os.unlink(comptmp + 'c')
+        os.unlink(comptmp)
+    else:
+        # The easy way
+        proc = subprocess.Popen([container_exe, 'run'] + CONTAINER_EXE_EXTRA_ARGS[container_exe] +
+                                ['--rm', '--name', '{}'.format(outfile),
+                                 '-v', '{}:/indir:Z'.format(indir),
+                                 '-v', '{}:/outdir:Z'.format(os.getcwd()), '-w', '/outdir',
+                                 'python:{}'.format(fullver),
+                                 'python', '-c',
+                                 "import py_compile; py_compile.compile('/indir/{}', '{}')".format(infile, outfile)])
```

Comment:
I assume we have to set the working directory for Python in order for this to work...

Reply:
Ah, yeah. Another good catch! Updated.